docs/tutorials/shallow-join.rst

```diff
@@ -379,3 +386,6 @@ country's polygon by its average goldstein scale, indicating how events are cont
 during that time range.
 
 .. figure:: _static/img/tutorials/2016-07-26-shallow-join/aggregate-GDELT.png
+
+The final result of the analysis described in this tutorial is found in the Jupyter notebook: :download:`_static/shallow-join-gdelt.ipynb`.
+A static render of this notebook can be found on Github (https://github.com/locationtech/geomesa/docs/tutorials/_static/shallow-join-gdelt.ipynb).
```

Comment:
Paths here have to match where you put the notebook (e.g. ``_static/geomesa-examples-jupyter/shallow-join-gdelt.ipynb``).